crates/ruff_linter/src/rules/flake8_raise/rules/unnecessary_paren_on_raise_exception.rs

```diff
@@ -36,6 +36,9 @@ use crate::{AlwaysFixableViolation, Applicability, Edit, Fix};
 /// raise TypeError
 /// ```
 ///
+/// ## Fix Safety
+/// This rule's fix is marked as unsafe if removing the parentheses would also remove comments.
```

Comment:
It would be great if we could document the unsafety when `exception_type.is_some` too